activities/Stopwatch.activity/js/activity.js

```diff
@@ -243,5 +243,23 @@ define(["sugar-web/activity/activity","mustache", "sugar-web/env"], function (ac
         activity.getDatastoreObject().save();
 
     });
+    
+    // Switch to full screen when the full screen button is pressed
+  	document.getElementById("fullscreen-button").addEventListener('click', function() {
```

Comment:
Indentation issue: you've mixed TAB and SPACE. In this file it should be SPACE only.